packages/uniforms-bootstrap4/src/gridClassName.js

```diff
@@ -38,7 +65,14 @@ export default function gridClassName (grid, side) {
 
     // Example: {xs: 6, sm: 4, md: 3}
     if (typeof grid === 'object') {
-        return Object.keys(grid).map(size => gridClassNamePart(size, grid[size], side)).join(' ');
+
+        if (!grid['xs']) {
+            grid['xs'] = grid['sm'] || grid['md'] || grid['lg'] || grid['xl'];
+        }
```

Comment:
Two things here:
* Should we care about it?
* If so, use dot notation instead.

Reply:
Yes it is needed (becouse we use this `xs` to count the default `col-`.

Reply:
It doesn't mean it's needed - it means that it's required.